catalog/pipeline/fbc-release/0.7/README.md

```diff
@@ -0,0 +1,24 @@
+# FBC Release Pipeline
+
+FBC Release Pipeline
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| snapshot | The Snapshot in JSON format | No | - |
+| enterpriseContractPolicy | JSON representation of the EnterpriseContractPolicy | No | - |
+| fromIndex | Index image (catalog of catalogs) the FBC fragment will be added to | No | - |
+| overWriteFromIndex | Boolean indicating if the fromIndex should be overwritten | Yes | "false" |
+| binaryImage | OCP binary image to be baked into the index image | Yes | "" |
+| buildTags | List of additional tags the internal index image copy should be tagged with | Yes | "[]"|
+| addArches | List of arches the index image should be built for | Yes | "[]" |
+| signingUser | The User requesting the image inex signature | No | - |
+| signingConfigMapName | The ConfigMap Name required by the Pipeline | No | - |
+| requestUpdateTimeout | Max seconds to wait until the status is updated | Yes | - |
+| buildTimeoutSeconds | Max seconds to wait until the build finishes | Yes | - |
+
+## Changes since 0.6
```

Comment:
Can we keep all the changes and just add to them when we create new versions? I expect v0.7 to have 
changes since 0.6
changes since 0.5
changes since 0.4
...
If I am going from v0.2 to v0.7 I want to be able to go to one place to see what new things I am getting, not go look at 5 different READMEs

Reply:
I agree. Maybe we should add some automation to generate the README.md and the `run`and `sample` directories to standardize this. I will fix manually this and play a bit on this automation.